client-spring/src/test/java/com/netflix/conductor/client/spring/ExampleClient.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.client.spring;
+
+import com.netflix.conductor.client.worker.Worker;
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.TaskResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.util.StringUtils;
+
+import java.io.IOException;
+import java.util.Properties;
+
+@SpringBootApplication
+public class ExampleClient {
+
+    private static final Logger log = LoggerFactory.getLogger(ExampleClient.class);
+
+    public static void main(String[] args) throws IOException {
+        loadExternalConfig();
```

Comment:
This method is not needed. It's needed for the conductor server but not for the clients.